docs/podman-volume-create.1.md

```diff
@@ -0,0 +1,51 @@
+% podman-volume-create(1)
+
+## NAME
+podman\-volume\-create - Create a new volume
+
+## SYNOPSIS
+**podman volume create** [*options*]
+
+## DESCRIPTION
+
+Creates an empty volume and prepares it to be used by containers. The volume
+can be created with a specific name, if a name is not given a random name is
+generated. You can add metadata to the volume by using the **--label** flag and
+driver options can be set using the **--opt** flag.
+Note: changing drivers and full driver options are not yet fully supported.
```

Comment:
I'm not a big fan of "Note: not yet supported" kinds of notes in man pages.  They've a tendency of hanging around after things become more supported.  I'd rather list what is supported now and blog about what's not supported or is future development. @rhatdan, WDYT?

Reply:
Yes don't point out that your kid is ugly.  Let others decide.  :^)

Reply:
Okay, fixed. Didn't want us to end up with issues that these features are not there, hence the note.